test/core/utils/collect-results-from-frames.js

```diff
@@ -37,6 +37,37 @@ describe('axe.utils.collectResultsFromFrames', function () {
     frame.src = '../mock/frames/results-timeout.html';
     fixture.appendChild(frame);
 
+  });
+
+  it('should override the timeout with `options.frameWaitTime`, if provided', function (done) {
+    var orig = window.setTimeout;
+    window.setTimeout = function (fn, to) {
+      if (to === 90000) {
+        assert.ok('timeout set');
+        fn();
+      } else { // ping timeout
+        return orig(fn, to);
+      }
+      return 'cats';
+    };
+
+    var frame = document.createElement('iframe');
+    frame.addEventListener('load', function () {
+      var context = new Context(document);
+      var params = { frameWaitTime: 90000 };
+      axe.utils.collectResultsFromFrames(context, params, 'stuff', 'morestuff', noop,
```

Comment:
Is there a way to also cover this in an integration test, so we can make sure it works from axe.run?

Reply:
specifically, across iframes

Reply:
Good idea. I've added an integration test.